packages/census/scripts/modify-snarkjs-package.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+
+let packagePath = path.dirname(require.resolve("snarkjs"));
+
+while(!fs.existsSync(packagePath + '/package.json')) {
+    packagePath = packagePath.split('/');
+    packagePath.pop();
+    if(packagePath.length === 0) return;
```

Comment:
Having a `return` on a non-function (root code) would not work. I would display a `console.error(..)` and do a `process.exit(1)` right after